hub/instruction_handling/create/generalities/all.tex

```diff
@@ -234,24 +234,41 @@
 				\If \scenCreateExecutionNonEmptyInitCode     _{i} = 1  & \Then & \nextContextIsNew      _{i} \\
 			\end{array} \right.
 		\]
+\end{description}
+Let us define the following shorthands:
+\[
+	\left\{ \begin{array}{lcl}
+		\locCreateUpfrontGasCost & \define & G_\text{create} + \locMxpGas              \\
+		\locCreateFullGasCost    & \define & \locCreateUpfrontGasCost + \locStpGasPoop \\
+	\end{array} \right.
+\]
+\begin{description}
 	\item[\underline{Setting the \gasCost{}:}]
 		we impose
 		\begin{enumerate}
 			\item \If $\locStaticx + \locMxpx = 1$              \Then $\gasCost _{i} = 0$
-			\item \If $\locOogx + \scenCreateUnexceptional = 1$ \Then $\gasCost _{i} = G_\text{create} + \locMxpGas$
+			\item \If $\locOogx + \scenCreateUnexceptional = 1$ \Then $\gasCost _{i} = \locCreateUpfrontGasCost$
 		\end{enumerate}
 	\item[\underline{Setting the \gasNext{}:}]
 		we impose
-		\begin{enumerate}
-			\item \If $\scenCreateException                 _{i} = 1$ \Then $\gasNext_{i} = 0$ (\trash)
-			\item \If $\scenCreateNoContextChange           _{i} = 1$ \Then $\gasNext_{i} = \gasActual_{i} - \gasCost_{i}$
-			\item \If $\scenCreateExecutionNonEmptyInitCode _{i} = 1$ \Then
-				\[
-					\gasNext_{i} = \gasActual_{i} - 
-					\left[ \begin{array}{cl}
-						+ & \gasCost_{i} \\
-						+ & \locStpGasPoop  \\
-					\end{array} \right]
-				\]
-		\end{enumerate}
+		\[
+			\hspace*{-0.5cm}
+			\left\{ \begin{array}{lcl}
+				\If \scenCreateException                 _{i} = 1 & \Then & \gasNext _{i} = 0 ~ (\trash)                               \\
+				\If \scenCreateAbort                     _{i} = 1 & \Then & \gasNext _{i} = \gasActual _{i} - \locCreateUpfrontGasCost \\
+				\If \scenCreateFCond                     _{i} = 1 & \Then & \gasNext _{i} = \gasActual _{i} - \locCreateFullGasCost    \\
```

Comment:
This one here is the bug fix: `scenario/CREATE_FAILURE_CONDITION` had wrong gas comuptation